pkg/syncer/metadatasyncer.go

```diff
@@ -737,6 +740,20 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		if IsPodVMOnStretchSupervisorFSSEnabled {
 			log.Info("Syncing StoragePolicyUsage CRs with the kubernetes PVs that are in Bound state")
 			storagePolicyUsageCRSync(ctx, metadataSyncer)
+			// Trigger StoragePolicyQuota reconciler to handle add/delete event on StoragePolicyQuota.
+			storageQuotaEnablementTicker := time.NewTicker(common.DefaultFeatureEnablementCheckInterval)
+			defer storageQuotaEnablementTicker.Stop()
+			go func() {
+				for ; true; <-storageQuotaEnablementTicker.C {
+					if err := initStoragePolicyQuotaReconciler(ctx, metadataSyncer.cnsOperatorClient); err != nil {
```

Comment:
metadataSyncer.cnsOperatorClient is of type client.Client: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/af5362800c3feb68ae7c51ecd45e4868188e4e48/pkg/kubernetes/kubernetes.go#L178

metadataSyncer.SupervisorClient used in initVolumeHealthReconciler is of type clientset.Interface: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/af5362800c3feb68ae7c51ecd45e4868188e4e48/pkg/kubernetes/kubernetes.go#L148

Reply:
metadataSyncer.SupervisorClient is needed for processing inside the handler, however we needed metadataSyncer.cnsOperatorClient, but now removed it since we create the client locally.